tests/Hedgehog.Tests/Program.fs

```diff
@@ -0,0 +1,26 @@
+module Hedgehog.Tests.Main
+
+open Hedgehog
+
+#if FABLE_COMPILER
+open Fable.Mocha
+
+let allTests = testList "All tests" [
```

Comment:
I'd be okay with single-sourcing this (ie, not making it `FABLE_COMPILER` specific), then it'll require us to keep it in sync. Then a helper can be exposed in `TestHelpers` to keep this compiler switching stuff in one place.

Reply:
Done.